src/components/ChildBountyCreation/CreateChildBounty.tsx

```
@@ -0,0 +1,379 @@
+			console.log(err);
+		}
+		setAvailableBalance(balance);
+	};
+
+	const handleParentBountyChange = (index: number) => {
+		debounceCheckIsBountyActiveAndValid(index);
+		dispatch(childBountyCreationActions.setParentBountyIndex(Number(index)));
+	};
+
+	const getOnChainTx = async () => {
+		if (!api || !apiReady || parentBountyIndex == null || isNaN(parentBountyIndex)) return null;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)